test/indexTest.js

```diff
@@ -126,13 +126,15 @@ test('uint8ToHex', (t) => {
 })
 
 test('hexToUint8', (t) => {
-  t.plan(6)
+  t.plan(8)
   t.deepEqual(fromHex('00'), {0: 0})
   t.deepEqual(fromHex('1'), {0: 1})
   t.deepEqual(fromHex(''), {})
   t.deepEqual(fromHex('00ff'), {0: 0, 1: 255})
   t.deepEqual(fromHex('1e010203'), {0: 30, 1: 1, 2: 2, 3: 3})
-  t.throws(fromHex.bind(null, new Uint8Array(3)), /string/, 'errors if inputs are wrong type')
+  t.deepEqual(fromHex('1E010203'), {0: 30, 1: 1, 2: 2, 3: 3})
+  t.throws(fromHex.bind(null, new Uint8Array(3)), /must be a string/, 'errors if inputs are wrong type')
+  t.throws(fromHex.bind(null, '1e010203g'), /must be hex/, 'errors if input is not hex')
```

Comment:
Would it be worth explicitly adding a test case like the one you allude to in the docs (i.e. the `0x` prefix)?
```
  t.throws(fromHex.bind(null, '0x1ffb78'), /must be hex/, 'errors if input is not hex')
```

Reply:
will do!

Reply:
done